### DISCUSSION TITLE
[Feature Request]: MDU(Monika) 新番部分搜索功能

### DISCUSSION THREAD

**Discussion by xiaoxi654** [jxxghp/MoviePilot#3377] [讨论]
### 当前程序版本

v1.4.1

### 运行环境

Docker

### 功能改进类型

其他

### 功能改进

MDU 新番搜索是独立于其他的搜索页面的，搜索时只会搜索[种子列表](https://monikadesign.uk/torrents)的资源页面，而其中不包括[新番](https://monikadesign.uk/torrents/airing)。
这会导致在 MoviePilot 的搜索中无法搜索到新番分区的内容。
另外在添加订阅后，新番分区的内容无法被搜索到，导致无法完成订阅。

**Comment by AisukaYuki** [comment]
瞎改改MoviePilot的莫妮卡默认解析，改成了默认解析新番页面。加密成base64放到插件 **自定义索引站点 - 配置**
就能再订阅刷新追番的时候用莫妮卡啦。搜索也能搜索到新番。~~（虽然用自定义RSS能实现，qb rss加规则也行，重在折腾~~
要恢复原来的搜索,关掉插件，可能要**重启**一下。

#原配置
```json
{"id":"monikadesign","name":"莫妮卡","domain":"https://monikadesign.uk/","encoding":"UTF-8","public":false,"proxy":false,"search":{"paths":[{"path":"torrents/airing?perPage=100&name={keyword}","method":"get"}]},"browse":{"path":"torrents?perPage=100&page={page}","start":1},"torrents":{"list":{"selector":"table#torrent-list-table > tbody > tr"},"fields":{"id":{"selector":"a.view-torrent.torrent-listings-name","attribute":"href","filters":[{"name":"re_search","args":["\\d+",0]}]},"title":{"selector":"a.view-torrent.torrent-listings-name"},"description":{"selector":"span.view-torrent.torrent-listings-subhead > b"},"details":{"selector":"a.view-torrent.torrent-listings-name","attribute":"href"},"download":{"selector":"a[href*=\"/torrents/download/\"]","attribute":"href"},"date_elapsed":{"selector":"td.torrent-listings-age > span.badge-extra"},"date":{"text":"{% if fields['date_elapsed'] or fields['date_added'] %}{{ fields['date_elapsed'] if fields['date_elapsed'] else fields['date_added'] }}{% else %}now{% endif %}","filters":[{"name":"dateparse","args":"%Y-%m-%d %H:%M:%S"}]},"size":{"selector":"td.torrent-listings-size > span"},"seeders":{"selector":"td.torrent-listings-seeders > a > span"},"leechers":{"selector":"td.torrent-listings-leechers > a > span"},"grabs":{"selector":"td.torrent-listings-completed > a > span"},"downloadvolumefactor":{"case":{"span.torrent-listings-freeleech":0,"*":1}},"uploadvolumefactor":{"case":{"*":1}}}},"price":{"FREE":["//span[@class='badge-extra'][contains(@data-original-title,'Free')]"],"2XFREE":[],"HR":[],"PEER_COUNT":["//a[@class='badge-extra text-green'][contains(@data-original-title,'做种')]"]},"system_type":""}
```

#加密配置，粘贴到自定义索引插件
```
monikadesign.uk|eyJpZCI6Im1vbmlrYWRlc2lnbiIsIm5hbWUiOiLojqvlpq7ljaEiLCJkb21haW4iOiJodHRwczovL21vbmlrYWRlc2lnbi51ay8iLCJlbmNvZGluZyI6IlVURi04IiwicHVibGljIjpmYWxzZSwicHJveHkiOmZhbHNlLCJzZWFyY2giOnsicGF0aHMiOlt7InBhdGgiOiJ0b3JyZW50cy9haXJpbmc/cGVyUGFnZT0xMDAmbmFtZT17a2V5d29yZH0iLCJtZXRob2QiOiJnZXQifV19LCJicm93c2UiOnsicGF0aCI6InRvcnJlbnRzP3BlclBhZ2U9MTAwJnBhZ2U9e3BhZ2V9Iiwic3RhcnQiOjF9LCJ0b3JyZW50cyI6eyJsaXN0Ijp7InNlbGVjdG9yIjoidGFibGUjdG9ycmVudC1saXN0LXRhYmxlID4gdGJvZHkgPiB0ciJ9LCJmaWVsZHMiOnsiaWQiOnsic2VsZWN0b3IiOiJhLnZpZXctdG9ycmVudC50b3JyZW50LWxpc3RpbmdzLW5hbWUiLCJhdHRyaWJ1dGUiOiJocmVmIiwiZmlsdGVycyI6W3sibmFtZSI6InJlX3NlYXJjaCIsImFyZ3MiOlsiXFxkKyIsMF19XX0sInRpdGxlIjp7InNlbGVjdG9yIjoiYS52aWV3LXRvcnJlbnQudG9ycmVudC1saXN0aW5ncy1uYW1lIn0sImRlc2NyaXB0aW9uIjp7InNlbGVjdG9yIjoic3Bhbi52aWV3LXRvcnJlbnQudG9ycmVudC1saXN0aW5ncy1zdWJoZWFkID4gYiJ9LCJkZXRhaWxzIjp7InNlbGVjdG9yIjoiYS52aWV3LXRvcnJlbnQudG9ycmVudC1saXN0aW5ncy1uYW1lIiwiYXR0cmlidXRlIjoiaHJlZiJ9LCJkb3dubG9hZCI6eyJzZWxlY3RvciI6ImFbaHJlZio9XCIvdG9ycmVudHMvZG93bmxvYWQvXCJdIiwiYXR0cmlidXRlIjoiaHJlZiJ9LCJkYXRlX2VsYXBzZWQiOnsic2VsZWN0b3IiOiJ0ZC50b3JyZW50LWxpc3RpbmdzLWFnZSA+IHNwYW4uYmFkZ2UtZXh0cmEifSwiZGF0ZSI6eyJ0ZXh0IjoieyUgaWYgZmllbGRzWydkYXRlX2VsYXBzZWQnXSBvciBmaWVsZHNbJ2RhdGVfYWRkZWQnXSAlfXt7IGZpZWxkc1snZGF0ZV9lbGFwc2VkJ10gaWYgZmllbGRzWydkYXRlX2VsYXBzZWQnXSBlbHNlIGZpZWxkc1snZGF0ZV9hZGRlZCddIH19eyUgZWxzZSAlfW5vd3slIGVuZGlmICV9IiwiZmlsdGVycyI6W3sibmFtZSI6ImRhdGVwYXJzZSIsImFyZ3MiOiIlWS0lbS0lZCAlSDolTTolUyJ9XX0sInNpemUiOnsic2VsZWN0b3IiOiJ0ZC50b3JyZW50LWxpc3RpbmdzLXNpemUgPiBzcGFuIn0sInNlZWRlcnMiOnsic2VsZWN0b3IiOiJ0ZC50b3JyZW50LWxpc3RpbmdzLXNlZWRlcnMgPiBhID4gc3BhbiJ9LCJsZWVjaGVycyI6eyJzZWxlY3RvciI6InRkLnRvcnJlbnQtbGlzdGluZ3MtbGVlY2hlcnMgPiBhID4gc3BhbiJ9LCJncmFicyI6eyJzZWxlY3RvciI6InRkLnRvcnJlbnQtbGlzdGluZ3MtY29tcGxldGVkID4gYSA+IHNwYW4ifSwiZG93bmxvYWR2b2x1bWVmYWN0b3IiOnsiY2FzZSI6eyJzcGFuLnRvcnJlbnQtbGlzdGluZ3MtZnJlZWxlZWNoIjowLCIqIjoxfX0sInVwbG9hZHZvbHVtZWZhY3RvciI6eyJjYXNlIjp7IioiOjF9fX19LCJwcmljZSI6eyJGUkVFIjpbIi8vc3BhbltAY2xhc3M9J2JhZGdlLWV4dHJhJ11bY29udGFpbnMoQGRhdGEtb3JpZ2luYWwtdGl0bGUsJ0ZyZWUnKV0iXSwiMlhGUkVFIjpbXSwiSFIiOltdLCJQRUVSX0NPVU5UIjpbIi8vYVtAY2xhc3M9J2JhZGdlLWV4dHJhIHRleHQtZ3JlZW4nXVtjb250YWlucyhAZGF0YS1vcmlnaW5hbC10aXRsZSwn5YGa56eNJyldIl19LCJzeXN0ZW1fdHlwZSI6IiJ9Cg==
```
![photo_2_2023-12-20_17-43-42](https://github.com/jxxghp/MoviePilot/assets/17586327/141f74b8-fae8-4cd1-9bbc-054ab88a77c5)
![photo_3_2023-12-20_17-43-42](https://github.com/jxxghp/MoviePilot/assets/17586327/95040514-c329-4540-951d-5d5c7ddb6172)
![photo_1_2023-12-20_17-43-42](https://github.com/jxxghp/MoviePilot/assets/17586327/7246da8b-7ea2-4283-91c6-eb2cbbfd99e1)

**Comment by kingtalk** [comment]
> 瞎改改MoviePilot的莫妮卡默认解析，改成了默认解析新番页面。加密成base64放到插件 **自定义索引站点 - 配置** 就能再订阅刷新追番的时候用莫妮卡啦。搜索也能搜索到新番。~（虽然用自定义RSS能实现，qb rss加规则也行，重在折腾~ 要恢复原来的搜索,关掉插件，可能要**重启**一下。
> 
> ```json
> {"id":"monikadesign","name":"莫妮卡","domain":"https://monikadesign.uk/","encoding":"UTF-8","public":false,"proxy":false,"search":{"paths":[{"path":"torrents/airing?perPage=100&name={keyword}","method":"get"}]},"browse":{"path":"torrents?perPage=100&page={page}","start":1},"torrents":{"list":{"selector":"table#torrent-list-table > tbody > tr"},"fields":{"id":{"selector":"a.view-torrent.torrent-listings-name","attribute":"href","filters":[{"name":"re_search","args":["\\d+",0]}]},"title":{"selector":"a.view-torrent.torrent-listings-name"},"description":{"selector":"span.view-torrent.torrent-listings-subhead > b"},"details":{"selector":"a.view-torrent.torrent-listings-name","attribute":"href"},"download":{"selector":"a[href*=\"/torrents/download/\"]","attribute":"href"},"date_elapsed":{"selector":"td.torrent-listings-age > span.badge-extra"},"date":{"text":"{% if fields['date_elapsed'] or fields['date_added'] %}{{ fields['date_elapsed'] if fields['date_elapsed'] else fields['date_added'] }}{% else %}now{% endif %}","filters":[{"name":"dateparse","args":"%Y-%m-%d %H:%M:%S"}]},"size":{"selector":"td.torrent-listings-size > span"},"seeders":{"selector":"td.torrent-listings-seeders > a > span"},"leechers":{"selector":"td.torrent-listings-leechers > a > span"},"grabs":{"selector":"td.torrent-listings-completed > a > span"},"downloadvolumefactor":{"case":{"span.torrent-listings-freeleech":0,"*":1}},"uploadvolumefactor":{"case":{"*":1}}}},"price":{"FREE":["//span[@class='badge-extra'][contains(@data-original-title,'Free')]"],"2XFREE":[],"HR":[],"PEER_COUNT":["//a[@class='badge-extra text-green'][contains(@data-original-title,'做种')]"]},"system_type":""}
> ```
> 
> ```
> monikadesign.uk|eyJpZCI6Im1vbmlrYWRlc2lnbiIsIm5hbWUiOiLojqvlpq7ljaEiLCJkb21haW4iOiJodHRwczovL21vbmlrYWRlc2lnbi51ay8iLCJlbmNvZGluZyI6IlVURi04IiwicHVibGljIjpmYWxzZSwicHJveHkiOmZhbHNlLCJzZWFyY2giOnsicGF0aHMiOlt7InBhdGgiOiJ0b3JyZW50cy9haXJpbmc/cGVyUGFnZT0xMDAmbmFtZT17a2V5d29yZH0iLCJtZXRob2QiOiJnZXQifV19LCJicm93c2UiOnsicGF0aCI6InRvcnJlbnRzP3BlclBhZ2U9MTAwJnBhZ2U9e3BhZ2V9Iiwic3RhcnQiOjF9LCJ0b3JyZW50cyI6eyJsaXN0Ijp7InNlbGVjdG9yIjoidGFibGUjdG9ycmVudC1saXN0LXRhYmxlID4gdGJvZHkgPiB0ciJ9LCJmaWVsZHMiOnsiaWQiOnsic2VsZWN0b3IiOiJhLnZpZXctdG9ycmVudC50b3JyZW50LWxpc3RpbmdzLW5hbWUiLCJhdHRyaWJ1dGUiOiJocmVmIiwiZmlsdGVycyI6W3sibmFtZSI6InJlX3NlYXJjaCIsImFyZ3MiOlsiXFxkKyIsMF19XX0sInRpdGxlIjp7InNlbGVjdG9yIjoiYS52aWV3LXRvcnJlbnQudG9ycmVudC1saXN0aW5ncy1uYW1lIn0sImRlc2NyaXB0aW9uIjp7InNlbGVjdG9yIjoic3Bhbi52aWV3LXRvcnJlbnQudG9ycmVudC1saXN0aW5ncy1zdWJoZWFkID4gYiJ9LCJkZXRhaWxzIjp7InNlbGVjdG9yIjoiYS52aWV3LXRvcnJlbnQudG9ycmVudC1saXN0aW5ncy1uYW1lIiwiYXR0cmlidXRlIjoiaHJlZiJ9LCJkb3dubG9hZCI6eyJzZWxlY3RvciI6ImFbaHJlZio9XCIvdG9ycmVudHMvZG93bmxvYWQvXCJdIiwiYXR0cmlidXRlIjoiaHJlZiJ9LCJkYXRlX2VsYXBzZWQiOnsic2VsZWN0b3IiOiJ0ZC50b3JyZW50LWxpc3RpbmdzLWFnZSA+IHNwYW4uYmFkZ2UtZXh0cmEifSwiZGF0ZSI6eyJ0ZXh0IjoieyUgaWYgZmllbGRzWydkYXRlX2VsYXBzZWQnXSBvciBmaWVsZHNbJ2RhdGVfYWRkZWQnXSAlfXt7IGZpZWxkc1snZGF0ZV9lbGFwc2VkJ10gaWYgZmllbGRzWydkYXRlX2VsYXBzZWQnXSBlbHNlIGZpZWxkc1snZGF0ZV9hZGRlZCddIH19eyUgZWxzZSAlfW5vd3slIGVuZGlmICV9IiwiZmlsdGVycyI6W3sibmFtZSI6ImRhdGVwYXJzZSIsImFyZ3MiOiIlWS0lbS0lZCAlSDolTTolUyJ9XX0sInNpemUiOnsic2VsZWN0b3IiOiJ0ZC50b3JyZW50LWxpc3RpbmdzLXNpemUgPiBzcGFuIn0sInNlZWRlcnMiOnsic2VsZWN0b3IiOiJ0ZC50b3JyZW50LWxpc3RpbmdzLXNlZWRlcnMgPiBhID4gc3BhbiJ9LCJsZWVjaGVycyI6eyJzZWxlY3RvciI6InRkLnRvcnJlbnQtbGlzdGluZ3MtbGVlY2hlcnMgPiBhID4gc3BhbiJ9LCJncmFicyI6eyJzZWxlY3RvciI6InRkLnRvcnJlbnQtbGlzdGluZ3MtY29tcGxldGVkID4gYSA+IHNwYW4ifSwiZG93bmxvYWR2b2x1bWVmYWN0b3IiOnsiY2FzZSI6eyJzcGFuLnRvcnJlbnQtbGlzdGluZ3MtZnJlZWxlZWNoIjowLCIqIjoxfX0sInVwbG9hZHZvbHVtZWZhY3RvciI6eyJjYXNlIjp7IioiOjF9fX19LCJwcmljZSI6eyJGUkVFIjpbIi8vc3BhbltAY2xhc3M9J2JhZGdlLWV4dHJhJ11bY29udGFpbnMoQGRhdGEtb3JpZ2luYWwtdGl0bGUsJ0ZyZWUnKV0iXSwiMlhGUkVFIjpbXSwiSFIiOltdLCJQRUVSX0NPVU5UIjpbIi8vYVtAY2xhc3M9J2JhZGdlLWV4dHJhIHRleHQtZ3JlZW4nXVtjb250YWlucyhAZGF0YS1vcmlnaW5hbC10aXRsZSwn5YGa56eNJyldIl19LCJzeXN0ZW1fdHlwZSI6IiJ9Cg==
> ```
> 
> ![photo_2_2023-12-20_17-43-42](https://private-user-images.githubusercontent.com/17586327/291850159-141f74b8-fae8-4cd1-9bbc-054ab88a77c5.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUzNjk5NTUsIm5iZiI6MTcwNTM2OTY1NSwicGF0aCI6Ii8xNzU4NjMyNy8yOTE4NTAxNTktMTQxZjc0YjgtZmFlOC00Y2QxLTliYmMtMDU0YWI4OGE3N2M1LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE2VDAxNDczNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI1NTY5ODY0NmEzN2ZkZTE3NzdkNmVlNDJiZmYxMjdhMDUzMTk4Zjg0YjNmN2QwYTQwNzg4YzQ2ZGNkZDg4ZTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.GSGubWvbNpJT4a7hX3xsWj3ujEkT8sNx5dvy3dxkdZo) ![photo_3_2023-12-20_17-43-42](https://private-user-images.githubusercontent.com/17586327/291850171-95040514-c329-4540-951d-5d5c7ddb6172.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUzNjk5NTUsIm5iZiI6MTcwNTM2OTY1NSwicGF0aCI6Ii8xNzU4NjMyNy8yOTE4NTAxNzEtOTUwNDA1MTQtYzMyOS00NTQwLTk1MWQtNWQ1YzdkZGI2MTcyLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE2VDAxNDczNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYwYzQ2OTZiZGVjMjljOGYwMTI0NjhmZDBjMjM5NmNmZmRjMjFiNjY2NjFhMzhiNTgyMGM5YjEzZjBjMTgxMmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.LINIECCNifpDyvhjN__zoxhBtsVnrfYgupKbesPekgI) ![photo_1_2023-12-20_17-43-42](https://private-user-images.githubusercontent.com/17586327/291850191-7246da8b-7ea2-4283-91c6-eb2cbbfd99e1.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUzNjk5NTUsIm5iZiI6MTcwNTM2OTY1NSwicGF0aCI6Ii8xNzU4NjMyNy8yOTE4NTAxOTEtNzI0NmRhOGItN2VhMi00MjgzLTkxYzYtZWIyY2JiZmQ5OWUxLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE2VDAxNDczNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEwNTIzZTQ3YmM3MTk0MDY4NmFkNTI2NjBhNzBlY2RiOTE3Mzg4MTlmOTQzNDcxODUxMDVlZWZlN2JlN2RjZDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.5CHoNBlY1YQPyXOW7RHaTFy42bGwDU11EvrZ9sqqwZk)

请问这两行代码是在哪里输入？ssh吗？
